vtexdocs/devportal
# Fix/Layout Sidebar

#### What is the purpose of this pull request?

To add an option to the layout component for rendering the sidebar conditionally.

#### What problem is this solving?

The landing page (and potentially other pages) shouldn't have a sidebar.

#### How should this be manually tested?

Go to this page and check it doesn't show the sidebar. Also, try visiting other pages in the application and asserting the sidebar shows when it is supposed to and there are no duplicate sidebars.

#### Types of changes
